pallets/crowdloans/src/lib.rs

```diff
@@ -1368,5 +1504,11 @@ pub mod pallet {
             ));
             Ok(())
         }
+
+        // Returns contributions for ChildStorageKinds
+        fn get_contributions(vault: &mut Vault<T>, kind: ChildStorageKind) -> u128 {
```

Comment:
`get_contributions` can be rename to `has_childstorage`, then you can check all together if any childstorage has contribution, if yes then return true, if no then return false